locales/en.json

```diff
@@ -1,15 +1,15 @@
 {
 	"Collaborative markdown notes": "Collaborative markdown notes",
-	"Realtime collaborative markdown notes on all platforms.": "Realtime collaborative markdown notes on all platforms.",
-	"Best way to write and share your knowledge in markdown.": "Best way to write and share your knowledge in markdown.",
+	"Realtime collaborative markdown notes on all platforms.": "Callaborate on markdown notes on all platforms.",
```

Comment:
So here is trying to emphasize the "realtime".
But I can see using a verb on head feels better.